.github/jitx-client/install.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+set -e
+
+chmod +x /root/.jitx/current/jitx # artifact caching loses permissions somewhere
```

Comment:
an alternative (hack) is to `tar` or `zip` the artifact separately, which handles the whole bundle.

Reply:
Do you plan on publishing another image with jitx completely installed so that I don't need this install.sh?

Reply:
The most recent push of the stable client should avoid this

Reply:
dev, latest or both tags of jitx-client?

Reply:
dev

Reply:
Great, the CI always pulls the last image so I just removed my jitx installation steps and it stills runs great, thank you!